showcases/chess.md

```diff
@@ -0,0 +1,234 @@
+---
+database-plugin: basic
+title: Sauronchess
+tags:
+  - puzzle
+date: 2025-02-02T00:00:00.000+01:00
+targetLink: https://github.com/FSaurenbach/sauronchess
+icon: /assets/images/portfolio/chess.png
+image: /assets/images/portfolio/chess.png
+showcase: true
+---
+
+```yaml:dbfolder
```

Comment:
Can you delete all this `yaml:dbfolder` thing? The body of this document should be just a description of the project